doc/release_notes/v2.1.0.rst

```diff
@@ -3,6 +3,7 @@ Version 2.1.0
 
 Changes
 -------
+* Dropped support for Python 3.5 (only Python 3.6+ supported)
```

Comment:
EOL for Python 3.5 is 2020-09-13 - we may be a bit early...

Reply:
I have no problems with that. I also like f-strings and type hints, and you are right - the rollout of new versions is faster now. So, yes, 👍

Reply:
Also means I don't need to bother with wrapping `str` around `pathlib.Path` objects. :).